app/src/main/kotlin/com/wire/android/feature/auth/registration/personal/email/CreatePersonalAccountEmailFragment.kt

```diff
@@ -0,0 +1,12 @@
+package com.wire.android.feature.auth.registration.personal.email
+
+import androidx.fragment.app.Fragment
+import com.wire.android.R
+
+class CreatePersonalAccountEmailFragment :
+    Fragment(R.layout.fragment_create_personal_account_email) {
```

Comment:
If the line length is too long, could be wrap the parameter? 

```
 Fragment(
    R.layout.fragment_create_personal_account_email
 )

```

I personally think it looks nicer than moving the whole` Fragment(...)` declaration onto another line.

Reply:
done. I better add detekt soon...